### ISSUE TITLE
Crash and a bug with the conversion monitor

### ISSUE THREAD

**Issue by jk1895** [PrototypeTrousers/Applied-Energistics-2#47]
Hello!

**Describe the bug**
The game crashes when using the conversion monitor. It also shows only 64 items, although there are more in the system. It has not voided the items, if I have seen it correctly.

**To Reproduce**
_Program_ the monitor. I used energetic alloy ingots.
Lock it.
Try to take items out and put them back in (Through the monitor.). -> Crash

**Additional context**
[crash-2021-08-16_21.33.56-server.txt](https://github.com/PrototypeTrousers/Applied-Energistics-2/files/6994994/crash-2021-08-16_21.33.56-server.txt)
[crash-2021-08-16_21.34.02-server.txt](https://github.com/PrototypeTrousers/Applied-Energistics-2/files/6994995/crash-2021-08-16_21.34.02-server.txt)

**Environment**
I play Omnifactory STE.

- Minecraft Version: 1.12.2 
- AE2 Version: appliedenergistics2-rv6-stable-7.omni-fixes-v46f.jar
- Forge Version: 14.23.5.2855

**Comment by PrototypeTrousers** [comment]
v46g should fix this!

**Comment by jk1895** [comment]
Unfortunately, it still crashes.
I have placed the conversion monitor.
Set an ingot with right click.
It shows 0, although it should be 250.
Then used bare hand and shift-right click to lock the monitor.
Then I tried placing the one ingot on the network with a right click of the monitor.
The game crashed.

[crash-2021-08-17_10.24.11-server.txt](https://github.com/PrototypeTrousers/Applied-Energistics-2/files/6998350/crash-2021-08-17_10.24.11-server.txt)
